### PR TITLE
update crowdin source to be locales/base/*.json

### PR DESCRIPTION
**Pull request by kathaypacific**
### Description

In Crowdin, English is now enabled as both a source and target language. This is because Crowdin only includes target languages in the OTA distribution, and we had no choice but to make this config change in Crowdin. Since then, we have noticed English translation updates made in Crowdin being overwritten because of bi-directional sync between Github and Crowdin. 

In this PR, update the Crowdin source to be `locales/base/*.json`. English will still be a target language, so after merging this PR Crowdin should create a new folder `locales/en/*.json`. While these two folders will hold mostly duplicate content, the result is no more bi-directional sync by separating the target and source English files.

Functionally the app should be mostly unaffected, and we should treat `locales/base/*.json` as we treat `locales/base/en-US/*.json` now. 

### Other changes

n/a

### Tested

Manually clicking around the app

### How others should test

Same

### CODE REVIEW

File: packages/mobile/scripts/translateFile.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const fileName = process.argv[2]
 const googleApiToken = process.argv[3]
 console.info(`Translating file: ${fileName}`)
 
-const json = fs.readFileSync(`../locales/en-US/${fileName}`)
+const json = fs.readFileSync(`../locales/base/${fileName}`)
```
Comment (kathaypacific) [inline packages/mobile/scripts/translateFile.js:10]: i wasn't sure this guy and verify_locales were still being used, but i updated them just in case

Comment (jeanregisser) [reply packages/mobile/scripts/translateFile.js:10]: Ah yes, this script should be deleted in another PR.

**Review by jeanregisser — APPROVED** [review]
Great! 👍
Can we merge #1514 before this?

**Comment by kathaypacific** [comment]
> Great! 👍 Can we merge #1514 before this?

Yup i had queued that guy for merging yesterday! It's finally gone in so I'll automerge this one :)

**Comment by ValoraQA** [comment]
Hey @MuckT, We have verified this task on Android Alfajores Nightly build & iOS Test flight Alfajores Nightly build V1.23.0 & observed that few components & text are not seen translated.

**Bug Link:**
[Bug1](https://github.com/valora-inc/wallet/issues/1691), [Bug2](https://github.com/valora-inc/wallet/issues/1693), [Bug3](https://github.com/valora-inc/wallet/issues/1694), [Bug4](https://github.com/valora-inc/wallet/issues/1695), [Bug5](https://github.com/valora-inc/wallet/issues/1696)

**Query link:**
[Query1](https://github.com/valora-inc/wallet/issues/1692)

**Devices:**  iPhone 12(14.7.1), Samsung Galaxy A8+(8), iPhone 13 mini (15.1.1), OnePlus 7T (11), Redmi Note 8 (9.0)
CC: @kathaypacific 
Thanks..!!

**Pull request merged**